core/inject.js

```diff
@@ -61,6 +61,10 @@ Blockly.inject = function(container, opt_options) {
     (/** @type {!Blockly.BlocklyOptions} */ ({})));
   var subContainer = document.createElement('div');
   subContainer.className = 'injectionDiv';
+  subContainer.tabIndex = 0;
+  Blockly.utils.aria.setState(subContainer,
+      Blockly.utils.aria.State.LABEL, Blockly.Msg.WORKSPACE_ARIA_LABEL);
```

Comment:
Blockly.Msg is of type object, so for advanced compilation, you'll need to use string lookup instead ``Blockly.Msg['WORKSPACE_ARIA_LABEL']``